### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**

### CODE REVIEW

**Review by abhijitdotsharma — APPROVED** [review]
Try to make smaller pr's, will be easy to review.

File: CSS/install.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
 @import url("base/reset.css");
 @import url("base/color.css");
-@import url("base/nav.css");
\ No newline at end of file
+@import url("base/nav.css");
+.installation-container {
+    box-shadow: 1px 1px 2px 2px var(--dark-primary-text);
```
Comment (mariyasada) [inline CSS/install.css:5]: keep space between import statement and class

File: CSS/lists.css [hunk 2]
```diff
@@ -1,3 +1,30 @@
 @import url("base/reset.css");
 @import url("base/color.css");
-@import url("base/nav.css");
\ No newline at end of file
+@import url("base/nav.css");
+.list-container {
+    box-shadow: 1px 1px 2px 2px var(--dark-primary-text);
+    width: 90%;
+}
```
Comment (mariyasada) [inline CSS/lists.css:7]: same here

**Review by mariyasada — APPROVED** [review]
look at the comments

**Pull request merged**